receiver/googlecloudspannerreceiver/metadata.yaml

```diff
@@ -0,0 +1,14 @@
+type: googlecloudspanner
+
+status:
+  class: receiver
+  stability:
+    beta: [metrics]
+  distributions: [contrib]
```

Comment:
Are you planning to create a pull request for [Releases](https://github.com/open-telemetry/opentelemetry-collector-releases/) repo to re-add the component to the Contrib distro?

Reply:
yep, i'll do that.  It looks like it is currently referencing the 112 release, though, which doesn't include this receiver, so i'm not sure if it will be able to be merged right away.